packages/xo-web/src/xo-app/user/index.js

```diff
@@ -282,6 +288,83 @@ const SshKeys = addSubscriptions({
   )
 })
 
+// ===================================================================
+const COLUMNS_AUTH_TOKENS = [
+  {
+    itemRenderer: ({ id }) => (
+      <Copiable tagName='pre' data={id}>
+        {id.slice(0, 5)}…
+      </Copiable>
+    ),
+    name: _('authToken'),
+  },
+  {
+    itemRenderer: token => (
+      <Text value={token.description} onChange={description => editAuthToken({ ...token, description })} />
```

Comment:
[`Text` requires a string `value`](https://github.com/vatesfr/xen-orchestra/blob/ab9970d2157c72bd36b839dd32fc88e48726b2a0/packages/xo-web/src/common/editable/index.js#L219) but `token.description` can be `undefined`.